test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1524,6 +1529,133 @@ static public void testCheckCastRefClassOnNull() throws Throwable {
 		checkCastRefClassOnNull.invoke();
 	}
 
+
+	// The three following tests can be used to verify that flattened value types in arrays are handled properly by the JIT and GC. In the
+	// current state, these tests should pass when flattened is disabled but failed when it is enabled. The JIT does not have adequate 
+	// support so until that is added, these tests can be run with the -Xint option enabled to disable to JIT compiler.
+	// /*
+	//  * Maintain a buffer of flattened arrays with single aligned valuetypes while keeping a certain amount of classes alive at any 
+	//  * single time. This forces the GC to unload the classes.
+	//  */
+	// @Test(priority=4)
+	// static public void testPersistantValueWithSingleAlignment() throws Throwable {
+	// 	String fields[] = {
+	// 		"tri:QTriangle2D;:value",
+	// 		"point:QPoint2D;:value",
+	// 		"line:QFlattenedLine2D;:value",
+	// 		"i:QValueInt;:value",
+	// 		"f:QValueFloat;:value",
+	// 		"tri2:QTriangle2D;:value"};
+	// 	Class assortedValueWithSingleAlignmentClass = ValueTypeGenerator.generateValueClass("ValueTypeWithSingleAlignment", fields);
```

Comment:
Also, this class is being defined elsewhere, so you can turn that variable into a global static